### PR TITLE
doc, log: fix typo in wal doc, log

### PR DESCRIPTION
**Pull request by ben1009**
fix typo in wal doc

### CODE REVIEW

File: server/storage/wal/repair.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func Repair(lg *zap.Logger, dirpath string) bool {
 			}
 
 			if _, err = io.Copy(bf, f); err != nil {
-				lg.Warn("failed to copy", zap.String("from", f.Name()+".broken"), zap.String("to", f.Name()), zap.Error(err))
+				lg.Warn("failed to copy", zap.String("from", f.Name()), zap.String("to", f.Name()+".broken"), zap.Error(err))
```
Comment (ptabor) [inline server/storage/wal/repair.go:80]: Thank you for catching. 

Could you, please, use the temporary variable in all the 3 places:

```
 toName := f.Name() + ".broken"
```

Comment (ptabor) [reply server/storage/wal/repair.go]: Thank you.

**Review by ptabor — APPROVED** [review]

**Review by ptabor — APPROVED** [review]

**Review by spzala — COMMENTED** [review]
@ben1009 can you please squash commits to one? thanks!

**Comment by ben1009** [comment]
> squash commits to one

Maybe a stupid question. Is that configurable through Allow squash merging?

**Comment by ben1009** [comment]
> @ben1009 can you please squash commits to one? thanks!

done

**Comment by ben1009** [comment]
> > @ben1009 can you please squash commits to one? thanks!
> 
> done

@spzala could you help take a look, thanks

**Pull request merged**